sklearn/mixture/gmm.py

```diff
@@ -64,7 +64,9 @@ def log_multivariate_normal_density(X, means, covars, covariance_type='diag'):
     return log_multivariate_normal_density_dict[covariance_type](
         X, means, covars)
 
-
+@deprecated("The function sample_gaussian is deprecated and "
```

Comment:
Please add the version they were deprecated in to the message (was deprecated in 0.18)